script/legacy/2019-05-14-armagaddon/delete-from-bq.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+## Affected period to delete is
+## 2019-05-04T11:00:00Z to 2019-05-11T11:00:00Z
+
+function delete_partitions() {
+    local TABLE=$1
+    bq rm -f -t "$TABLE"'$20190504'
+    bq rm -f -t "$TABLE"'$20190505'
+    bq rm -f -t "$TABLE"'$20190506'
+    bq rm -f -t "$TABLE"'$20190507'
+    bq rm -f -t "$TABLE"'$20190508'
+    bq rm -f -t "$TABLE"'$20190509'
+    bq rm -f -t "$TABLE"'$20190510'
+    bq rm -f -t "$TABLE"'$20190511'
+}
+
+## Tables populated by Dataflow jobs
+delete_partitions moz-fx-data-shar-nonprod-efed.activity_stream.impression_stats_v1
+delete_partitions moz-fx-data-shar-nonprod-efed.activity_stream.spoc_fills_v1
+
+## Tables copied from Parquet
+# Matches first appendix from the Google Doc (which lists AWS derived datasets to delete)
+delete_partitions moz-fx-data-derived-datasets.events_v1
+delete_partitions moz-fx-data-derived-datasets.experiments_v1
+delete_partitions moz-fx-data-derived-datasets.crash_summary_v1
+delete_partitions moz-fx-data-derived-datasets.first_shutdown_summary_v4
+# No landfill sample in BQ
+# No longitudinal in BQ
```

Comment:
I see `longitudinal_v20190511` and `longitudinal_20190504` listed in the bq schema browser